be/src/vec/aggregate_functions/aggregate_function_histogram.h

```diff
@@ -86,7 +92,9 @@ struct AggregateFunctionHistogramData {
     }
 
     void merge(const AggregateFunctionHistogramData& rhs) {
-        if (!rhs.max_num_buckets) {
+        // if rhs.max_num_buckets == 1, it means the input block for serialization is all null
```

Comment:
rhs.max_num_buckets == 1,  seems is means == -1?

Reply:
yes, i will fix